packages/server/src/queue/queue.entity.ts

```diff
@@ -74,8 +76,14 @@ export class QueueModel extends BaseEntity {
   isOpen: boolean;
 
   async checkIsOpen(): Promise<boolean> {
-    this.isOpen =
-      this.staffList && this.staffList.length > 0 && !this.isDisabled;
+    if (!this.staffList) {
+      console.error(ERROR_MESSAGES.queueController.missingStaffList, this.id);
```

Comment:
nit - we don't need to log this if we throw the exception right? we could include the `id` in the error message below